documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc

```diff
@@ -329,11 +329,11 @@ NOQuuuWuFMMMCCCCPPPPTTTTTTTTRRRR
 
 ^1^ Information on xref:raspberry-pi.adoc#otp-register-and-bit-definitions[programming the OTP bits].
 
-^2^ Warranty bit is never set on Pi4.
+^2^ Warranty bit is never set on Pi 4.
```

Comment:
We're going to try and remove all references to "Pi" it should in all cases be "Raspberry Pi"

Reply:
I've gone through the file and fixed up all occurrences of this.